syntax/index.html

```diff
@@ -498,12 +498,12 @@
         </div>
       </section>
       <section>
-        <h3>Site-wide Location Configuration</h3>
+        <h3>Default Locations and Site-wide Location Configuration</h3>
```

Comment:
This is a step in the right direction, but please call a spade a spade.  This is a "standard metadata filename" feature that, for *algorithmic* *convenience*, the WG has defined as though it is a default of the "site-wide location configuration" feature.  That's okay from an algorithmic perspective, but from the *user's* perspective it is a different feature.  For the vast majority of users, the "site-wide location configuration" feature is *completely* irrelevant, because it is not the feature that they will *choose* to use.  The TAG insisted that users have a *choice*, and the "site-wide location configuration" feature gives them that choice.  The TAG did *not* say that the WG should attempt to bias the user's choice by hiding the "standard metadata filename" feature, giving it a name that makes it harder to find, or pretending that it is an inseparable part of the "site-wide location configuration" feature.  It is not.

The working group has already satisfied its responsibility to the TAG by including the "site-wide location configuration" feature.   I strongly encourage the WG to now be as helpful as possible to the community by making these features as easy as possible for users to find and identify -- *especially* the features that users are *most* likely to use.